tests/test_basic.py

```diff
@@ -297,3 +297,71 @@ def test_mons_up_down(rook_cluster):
         check += 1
 
     assert mon_pods == mons
+
+
+def test_rbd_pvc(rook_cluster):
+    # create a CephBlockPool
+    output = rook_cluster.kubernetes.kubectl_apply(
+                os.path.join(
+                    rook_cluster.ceph_dir, 'csi/rbd/storageclass.yaml'))
+
+    if output[0] != 0:
+        pytest.fail("Could not create a CephBlockPool StorageClass")
+
+    # check if StorageClass is up and available
+    output = rook_cluster.kubernetes.kubectl('get sc rook-ceph-block')
+
+    if output[0] != 0:
+        pytest.fail("Could not find a CephBlockPool called StorageClass")
+
+    # create an rbd based PVC
+    output = rook_cluster.kubernetes.kubectl_apply(
+                os.path.join(rook_cluster.ceph_dir, 'csi/rbd/pvc.yaml'))
+
+    if output[0] != 0:
+        pytest.fail("Could not create a rbd-PVC")
+
+    pvc = rook_cluster.kubernetes.kubectl('get -o yaml pvc rbd-pvc')
```

Comment:
ditto

Reply:
You are correct.
I am using `wait_for_result instead` now.